permits/templates/permits/_permit_request_actions.html

```diff
@@ -54,12 +54,13 @@ <h2>{% trans "" %}</h2>
 
       {% endif %}
 
-
-          <li class="nav-item">
-          <a class="btn btn-primary btn-action nav-link" id="print-tab" data-toggle="tab" href="#print" role="tab" aria-controls="print">
-              {% trans "Imprimer" %}
-          </a>
-          </li>
+      {% if has_permission_to_classify %}
```

Comment:
Should't be `has_permission_to_classify_permit_request`?

Reply:
It's just the short name used to pass the variable to the context:

`has_permission_to_classify": services.has_permission_to_classify_permit_request`